src/main/resources/META-INF/chisel_at.cfg

```diff
@@ -0,0 +1 @@
+public-f net.minecraft.block.Block field_176227_L # blockState
```

Comment:
Oh... did you just remove the reflection helper and make the field that we were after public?